### ISSUE TITLE
New adapters (eg. Matrix) roadmap

### ISSUE THREAD

**Issue by pataquets** [getgort/gort#207]
Just found gort and looks very interesting.
As I'm looking for using Matrix for my purposes, I could not find any support, which is understandable.
However, I've seen in the docs "more adapters in the works", but could not find any list of planned or WIP adapters to keep track. No issues, no list (even if not a priority now, which is fine).
Providing more visibility would be helpful.
Thanks.

**Comment by clockworksoul** [comment]
Hi @pataquets!

At this time there is no specific list, in large part because there just hasn't been much demand outside of Slack and Discord yet. 

Matrix is interesting though, and there does seem to be a Go client. If you think it'll be useful to you, I can add Matrix support to the roadmap!

**Comment by pataquets** [comment]
Indeed, I think it would be very interesting to support Matrix, since it's quite used and one of the main non-proprietary messaging alternatives (no pressure on that, thou).
What I'm missing too is a backend list (either available or planned) to provide visibility on roadmap and priorities. You mention adding it to a roadmap, but I didn't found any (for services). Making it more prominent for people interested in checking out (even with issue links to track and help it) would be great.
